internal/supervisor/installation.go

```diff
@@ -350,8 +353,51 @@ func (s *InstallationSupervisor) createClusterInstallation(cluster *model.Cluste
 		),
 	)
 	if cpuPercent > s.clusterResourceThreshold || memoryPercent > s.clusterResourceThreshold {
-		logger.Debugf("Cluster %s would exceed the cluster load threshold (%d%%): CPU=%d%%, Memory=%d%%", cluster.ID, s.clusterResourceThreshold, cpuPercent, memoryPercent)
-		return nil
+		if s.clusterResourceThresholdScaleValue != 0 &&
```

Comment:
I think we should flip this conditional so it's `if s.clusterResourceThresholdScaleValue == 0` and return early (the body of the `else` function) and then the body of the `if` can be outdented and the `else` block eliminated.